# This message serves as switch command for the rover locomotion control
Header header
uint16 SteerPower 					# Steering power switch
uint16 DrivePower 					# Driving power switch
uint16 Publisher					# Activate wheel encoder publisher
uint16 ZeroEncoders					# Zero wheel encoders
